src/event_loop.rs

```diff
@@ -8,25 +8,18 @@
 //!
```

Comment:
Nit: Might make sense to separate this file in two in a separate PR/commit, just to make the git history a bit easier to follow?